### PR TITLE
Feature/create new subscription

### PR DESCRIPTION
**Pull request by cblanco**
- New add subscription button
- Change the position where the JSON editor widget opens

### CODE REVIEW

File: src/js/main.js [hunk 1]
```diff
@@ -60,6 +60,18 @@
...
+            class: "se-btn-circle add-entity-button z-depth-3",
+            iconClass: "fa fa-plus",
+        });
+        this.create_entity_button.addEventListener('click', function (button) {
+            openEditorWidget.call(this, button, "create");
+            this.template_output.pushEvent(JSON.stringify(emptySubscription));
+        }.bind(this));
+
+        this.layout.center.appendChild(this.create_entity_button);
+        this.defaultEditorPos = {"y": 0,"x": 0,"width": 0,"height": 0,"top": 0,"right": 0,"bottom": 0,"left": 0};
```
Comment (aarranz) [inline src/js/main.js]: Please, remove this `defaultEditorPost` field. Thanks.

File: src/js/main.js [hunk 2]
```diff
@@ -124,7 +157,7 @@
 
     var openEditorWidget = function openEditorWidget(button, action) {
         if (this.editor_widget == null) {
-            this.editor_widget = mp.mashup.addWidget('CoNWeT/json-editor/1.0', {refposition: button.getBoundingClientRect()});
+            this.editor_widget = mp.mashup.addWidget('CoNWeT/json-editor/1.0', {refposition: this.defaultEditorPos});
```
Comment (aarranz) [inline src/js/main.js]: And revert this change 😉.

**Review by aarranz — CHANGES_REQUESTED** [review]

File: src/js/main.js (hunk 2 above)
Comment (aarranz) [reply src/js/main.js]: better, use directly the button without calling `getBoundingClientRect`

Comment (cblanco) [reply src/js/main.js]: Fixed in 433f4c6.

json-editor static version updated to v1.0.1
